css/platinui.win.css

```diff
@@ -224,5 +224,52 @@ html {
   bottom: 1px;
   left: 1px;
 }
+.ui-windows label {
```

Comment:
The `label` selection is very generic, it will conflict other labels, you should provide some class like `.radio`.